src/appleseed-max-impl/appleseedrenderer/appleseedrendererparamdlg.cpp

```diff
@@ -706,11 +707,15 @@ namespace
         {
             ReleaseISpinner(m_spinner_renderingthreads);
             ReleaseICustEdit(m_text_renderingthreads);
+            ReleaseICustEdit(m_text_render_stamp);
             m_rend_params->DeleteRollupPage(m_rollup);
         }
 
         virtual void init(HWND hwnd) override
         {
+            m_text_render_stamp = GetICustEdit(GetDlgItem(hwnd, IDC_TEXT_RENDER_STAMP));
```

Comment:
Same thing here, keep initialization order consistent with the settings order, data member orders, etc.